samples/deploy/crd/v1/ApisixConsumer.yaml

```diff
@@ -46,6 +46,8 @@ spec:
               properties:
                 ingressClassName:
                   type: string
+                description:
+                  type: string
```

Comment:
the apisix consumer description definition schema sets the maxLength of this to 256. https://github.com/apache/apisix/blob/1a45d1da896d22048b08b1bdc1218a84468a98a2/apisix/schema_def.lua#L109
I think it will be good to enforce it at this level as well.

Reply:
Sorry for the delay, change applied! Thanks @Revolyssup!